src/common/utils.js

```diff
@@ -25,13 +25,52 @@ import RequestTemplates from 'features/RequestForm/RequestTemplates'
 
 const { Item, SubMenu } = Menu
 
+export const getDataLocalStorage = type => {
+  try {
+    return JSON.parse(window.localStorage.getItem('currentUser'))[type]
+  } catch (err) {
+    return 'user'
+  }
+}
+
+const getRoleAccessLevel = role => {
+  switch (role) {
+    case 'admin':
+      return 1
+    case 'user':
+      return 0
+    default:
+      return 0
+  }
+}
+
 // Menu Creator
-export const MenuCreator = menuListItems =>
-  menuListItems.map(({ submenu, path, name, icon }) =>
-    submenu.length ? (
-      <SubMenu
-        key={shortId.generate()}
-        title={
+export const MenuCreator = menuListItems => {
+  const roleAccessLevel = getRoleAccessLevel(getDataLocalStorage('role'))
+
+  return menuListItems.map(({ submenu, path, name, icon, accessLevel }) =>
+    roleAccessLevel >= accessLevel ? (
```

Comment:
You could use here logical && operator and then ternary, to avoid nesting ternary and esLint doesn't complain